gap/oper.gi

```diff
@@ -270,6 +270,14 @@ InstallMethod(DigraphRemoveEdges, "for an immutable digraph and a list",
 [IsImmutableDigraph, IsList],
 {D, edges} -> MakeImmutable(DigraphRemoveEdges(DigraphMutableCopy(D), edges)));
 
+InstallMethod(DigraphRemoveEdge, "for an immutable digraph and an empty list",
+[IsImmutableDigraph, IsList and IsEmpty],
+{D, edge} -> D);
```

Comment:
I don't think you should add this method. It is undocumented and I think deliberately unsupported: `DigraphRemoveEdge` for a digraph and a list is documented for a _pair of vertices_. So I think the current behaviour is correct:
```
gap> DigraphRemoveEdge(ChainDigraph(2), []);
Error, the 2nd argument <edge> must be a list of length 2, at /Users/Wilf/gap/pkg/digraphs/gap/oper.gi:252 called from
DigraphRemoveEdge( DigraphMutableCopy( D ), edge
 ) at /Users/Wilf/gap/pkg/digraphs/gap/oper.gi:260 called from
<function "DigraphRemoveEdge for a immutable digraph and a list">( <arguments>\
 )
```

Reply:
This pull request was opened to respond to #260 however my version of digraphs was outdated upon writing this, after updating the update renders this pull request redundant and I believe resolves #260 aswell.